src/android/InAppBrowser.java

```diff
@@ -966,21 +1033,56 @@ public boolean onKey(View v, int keyCode, KeyEvent event) {
                     }
                 });
 
+                // Navigation bar layout
+                GradientDrawable navigationBarBorderBackground = new GradientDrawable();
+                navigationBarBorderBackground.setColor(toolbarColor);
+                navigationBarBorderBackground.setStroke(1, android.graphics.Color.parseColor("#E9E6E1"));
+                RelativeLayout navigationBar = new RelativeLayout(cordova.getActivity());
+                if(android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.JELLY_BEAN) {
+                    navigationBar.setBackgroundDrawable(navigationBarBorderBackground);
+                } else {
+                    navigationBar.setBackground(navigationBarBorderBackground);
+                }
+                navigationBar.setLayoutParams(new RelativeLayout.LayoutParams(LayoutParams.MATCH_PARENT, this.dpToPixels(44)));
+                navigationBar.setPadding(this.dpToPixels(2), this.dpToPixels(2), this.dpToPixels(2), this.dpToPixels(2));
+                navigationBar.setVerticalGravity(Gravity.CENTER);
+                navigationBar.setHorizontalGravity(Gravity.LEFT);
+
+                // Add page title for navigation bar
```

Comment:
I know in a lot of cases we've wanted to hide any indication of what page the user is on in the past. Is it possible to include another flag that will show/hide the page title in the navigation bar?

Reply:
Can you say more on what those cases are?

Reply:
Anything we don't want a user sharing the URL - deals only given to certain users, earn extra where we direct users to a specific URL to trigger payments, etc. But even if that's reduced to a page title, I could see not wanting to share the page title if it's some random survey provider's website (cc @rberman to confirm). Also thinking about cases like the Code for America Child Tax Credit pilot where I think they didn't want us to share any website info in the beginning as the website was live, but they only wanted a small group of Providers users to start out.

Reply:
The `navigationbar` option is a bit of a misnomer, it's more like `saveteamiab` because it displays a navigation bar, adds page title, and adds a reload button.

I think this code is complicated enough as it is. I can break it down further into `showpagetitle`, `navigationbar`, and `allowreload` options later.

Reply:
I renamed `navigationbar` to `pagetitleheader`